src/d3a/models/strategy/external_strategies/pv.py

```diff
@@ -13,7 +13,6 @@ class PVExternalMixin(ExternalMixin):
     """
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.pending_requests = []
```

Comment:
Why not  initiating the list here?

Reply:
I see now, you are calling _reject_all_pending_requests on every market cycle. Thanks!